app/src/main/res/values/strings.xml

```diff
@@ -639,4 +643,13 @@
     <string name="Search_Bar_Hint_Text">Search..</string>
     <string name="search_menu_title">Search</string>
 
+    <string name="learn_more">Learn More</string>
+
+    <string name="oscilloscope_description">Allows observation of varying \nsignal voltages</string>
+    <string name="multimeter_description">Measure voltages, currents \nand resistances</string>
+    <string name="logic_analyzer_description">Captures and displays signals \nfrom digital systems</string>
+    <string name="sensors_description">Allows logging of data returned \nby sensor connected</string>
+    <string name="wave_generator_description">Generates different waveforms \nand PWM signals</string>
+    <string name="power_source_description">Displays value returned by \nsensors connected to \n I2C input</string>
```

Comment:
Remove the space before "I2C"

Reply:
And don't manually create new line using \n instead fix the size of the text view.